mobile/src/main/java/org/openhab/habdroid/ui/PreferencesActivity.java

```diff
@@ -391,6 +446,18 @@ private void updateVibrationPreferenceIcon(Preference pref, Object newValue) {
                     ? R.drawable.ic_smartphone_grey_24dp : R.drawable.ic_vibration_grey_24dp);
         }
 
+        private void updateAlarmClockEnabledPreferenceIcon(Preference pref, Object newValue) {
+            boolean enabled = (boolean) newValue;
+            pref.setIcon(enabled ? R.drawable.ic_alarm_grey_24dp
+                    : R.drawable.ic_alarm_off_grey_24dp);
+        }
+
+        private void setEditorSummary(Preference pref, Object newValue) {
+            String itemName = (String) newValue;
```

Comment:
Generic method name, not-so-generic variable name ;-)

Reply:
Done